components/input-number/doc/index.en-US.md

```diff
@@ -19,25 +19,26 @@ import { NzInputNumberModule } from 'ng-zorro-antd/input-number';
 
 ### nz-input-number
 
-| property | description | type | default |
-| -------- | ----------- | ---- | ------- |
-| `[ngModel]` | current value, double binding | `number \| string`  \|  `string` | - |
-| `[nzAutoFocus]` | get focus when component mounted | `boolean` | `false` |
-| `[nzDisabled]` | disable the input | `boolean` | `false` |
-| `[nzMax]` | max value | `number` | `Infinity` |
-| `[nzMin]` | min value | `number` | `-Infinity` |
-| `[nzFormatter]` | Specifies the format of the value presented | `(value: number \| string) => string \| number` | - |
-| `[nzParser]` | Specifies the value extracted from nzFormatter | `(value: string) => string \| number` | `(value: string) => value.trim().replace(/。/g, '.').replace(/[^\w\.-]+/g, '')` |
-| `[nzPrecision]` | precision of input value | `number` | - |
+| property            | description | type | default |
```

Comment:
Unnecessary whitespace, please check your config of prettier :)

Reply:
done